tests/test_002-exclusive_jets.py

```diff
@@ -211,6 +211,225 @@ def test_exclusive_lund_declustering_multi():
+
+    softdrop_output = ak.zip(
+        {
+            "constituents": ak.Record(
+                {
+                    "px": [32.2, 32.45],
+                    "py": [64.21, 63.21],
+                    "pz": [543.34, 543.14],
+                    "E": [600.12, 599.56],
+                }
+            ),
+            "msoftdrop": 488.2395243115817,
+            "ptsoftdrop": 142.88274528437645,
+            "etasoftdrop": 2.726117171791057,
+            "phisoftdrop": 1.1012644074821902,
+            "Esoftdrop": 1199.6799999999998,
+            "pzsoftdrop": 1086.48,
+        },
+    )
+
+    is_close = ak.ravel(
+        ak.Array(
+            [
+                ak.isclose(
+                    softdrop_output.constituents.px,
+                    softdrop.constituents.px,
+                    rtol=1e-12,
+                    atol=0,
+                ),
+                ak.isclose(
+                    softdrop_output.constituents.py,
+                    softdrop.constituents.py,
+                    rtol=1e-12,
+                    atol=0,
+                ),
+                ak.isclose(
+                    softdrop_output.constituents.pz,
+                    softdrop.constituents.pz,
+                    rtol=1e-12,
+                    atol=0,
+                ),
+                ak.isclose(
+                    softdrop_output.constituents.E,
+                    softdrop.constituents.E,
+                    rtol=1e-12,
+                    atol=0,
+                ),
+                ak.isclose(
+                    ak.Array([softdrop_output.msoftdrop]),
+                    ak.Array([softdrop.msoftdrop]),
+                    rtol=1e-12,
+                    atol=0,
+                ),
+                ak.isclose(
+                    ak.Array([softdrop_output.ptsoftdrop]),
+                    ak.Array([softdrop.ptsoftdrop]),
+                    rtol=1e-12,
+                    atol=0,
+                ),
+                ak.isclose(
+                    ak.Array([softdrop_output.etasoftdrop]),
+                    ak.Array([softdrop.etasoftdrop]),
+                    rtol=1e-12,
+                    atol=0,
+                ),
+                ak.isclose(
+                    ak.Array([softdrop_output.phisoftdrop]),
+                    ak.Array([softdrop.phisoftdrop]),
+                    rtol=1e-12,
+                    atol=0,
+                ),
+                ak.isclose(
+                    ak.Array([softdrop_output.Esoftdrop]),
+                    ak.Array([softdrop.Esoftdrop]),
+                    rtol=1e-12,
+                    atol=0,
+                ),
+                ak.isclose(
+                    ak.Array([softdrop_output.pzsoftdrop]),
+                    ak.Array([softdrop.pzsoftdrop]),
+                    rtol=1e-12,
+                    atol=0,
+                ),
+            ]
+        )
+    )
+
+    assert ak.all(is_close)
+
+
+def test_exclsuive_jets_softdrop_grooming_multi():
```

Comment:
please add a dask test as well!

Reply:
Can you point me to an example of a dask test I can use as a starting point? I used the other single and multi events to make mine, but I'm not finding other dask tests I can use as a template.

Reply:
https://github.com/scikit-hep/fastjet/blob/main/tests/test_008-dask.py#L12

Reply:
I've added a dask test here: https://github.com/cmoore24-24/fastjet/blob/77b5363e526c53d946414f00d7ffcb8fad91f68e/tests/test_002-exclusive_jets.py#L321
Does this look about right?